.gitignore

```
@@ -19,3 +19,8 @@
 
 # Go workspace file
 go.work
+
+# Files generated by the operator.
+charts/kubearchive/crds/kubearchive.kubearchive.org_kubearchiveconfigs.yaml
+charts/kubearchive/templates/operator/role.yaml
+cmd/operator/api/v1alpha1/zz_generated.deepcopy.go
```

This is the pattern that is included with the operator-sdk gitignore which covers for extra files in case we need them.
```suggestion
!vendor/**/zz_generated.*
```

---

@rh-hemartin, if I make this change the file isn't ignored.

---

No problem then

---

@rh-hemartin , does it work for you?

---

I tested it and it does not work, the thing that works is:

```suggestion
zz_generated.*
```

Maybe we can go with this and the moment we vendor then we add the other line to include generated files in vendorized dependencies.